internal/exec/stages/disks/luks.go

```diff
@@ -300,6 +310,10 @@ func (s *stage) createLuks(config types.Config) error {
 					return fmt.Errorf("parsing tang URL: %v", err)
 				}
 				u.Path = path.Join(u.Path, "adv")
+				// If the advertisement is already set, we can assume we are in offline provisioning
+				if !util.NilOrEmpty(tang.Advertisement) {
+					continue
```

Comment:
As the comment says, we're just prefetching here to make sure the servers are all up.  The code is still doing a LUKS close/open cycle below, which will fail if the server is unreachable.

Reply:
Sigh; yeah, I saw this and forgot to do something about it because I went on a tangent about refactoring the code; I will have this resolved soon